core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.spec.ts

```diff
@@ -736,10 +736,10 @@ describe('Story editor navbar component', () => {
       storyEditorStateService, 'loadStory').and.returnValue();
     let alertServiceSpy = spyOn(
       alertsService, 'addInfoMessage').and.callThrough();
-    let check: number = 0;
+    let success: boolean = false;
```

Comment:
Still not named clearly enough. What exactly is succeeding?

Reply:
If this variable is true, then successCallback is called in the callFake() for saveStorySpy defined below, else errorCallback() is called.

Reply:
OK, can you name it (and the others) more clearly, please?

Reply:
Done